source/Generic/PlayState/PlayState.cs

```diff
@@ -242,6 +267,14 @@ public override void OnGameStarted(OnGameStartedEventArgs args)
                 return;
             }
 
+            playedDateTime = DateTime.Now;
+
+            showWindowsNotificationsStyle = false;
+            if (settings.Settings.GlobalShowWindowsNotificationsStyle)
+            {
+                showWindowsNotificationsStyle = true;
+            }
```

Comment:
A class could be created that has the info used by the extension, maybe like this

```csharp
    class PlaystateData
    {
        public id DatabaseId { get; set; }
        public datetime StartDate { get; set; }
        public Stopwatch Stopwatch { get; set; }
    }
```

This could replace the `List<Tuple<Guid, Stopwatch>>` list and use a `List<PlaystateData>` list instead to use in the extension. Maybe even put the revelant methods inside the class, like the ones to get the current playtime or so, but that's just an idea.

Reply:
I just followed the code as paused/resumed setting was being handled there too. But you are right, should be on the ShowNotification method.

I will see if I can make work the second comment.

Reply:
Done